packages/client-core/src/user/components/UserMenu/menus/AvatarSelectMenu.tsx

```diff
@@ -58,6 +60,8 @@ const AvatarMenu = () => {
   const authState = useHookstate(getMutableState(AuthState))
   const userId = authState.user?.id?.value
   const userAvatarId = useHookstate(getMutableState(AvatarState)[Engine.instance.userID].avatarID as EntityUUID)
+  const avatarLoading = useHookstate(false)
+  const isUserReady = getState(EngineState).userReady
```

Comment:
This needs to be `useHookstate(getMutableState(EngineState).userReady)`

Reply:
Implemented!